man/systemd-boot.xml

```diff
@@ -530,6 +530,25 @@
     url="https://systemd.io/BOOT_LOADER_INTERFACE">Boot Loader Interface</ulink>.</para>
   </refsect1>
 
+  <refsect1 id='uefi-shell-variables'>
+    <title>UEFI Shell Variables</title>
+
+    <para>The following <emphasis>UEFI Shell</emphasis> variables (vendor UUID
+    <literal>158def5a-f656-419c-b027-7a3192c079d2</literal>) are defined. They may be set using
```

Comment:
is there a spec somewher about this vendor uuid namespace? is it really ok to just squat the namespace?